aptoidepreferences/src/main/java/cm/aptoide/pt/preferences/managed/ManagerPreferences.java

```diff
@@ -56,7 +56,7 @@ public static boolean getGeneralDownloadsWifi(SharedPreferences sharedPreference
   }
 
   public static boolean getGeneralDownloadsMobile(SharedPreferences sharedPreferences) {
-    return sharedPreferences.getBoolean(ManagedKeys.GENERAL_DOWNLOADS_MOBILE, true);
+    return !sharedPreferences.getBoolean(ManagedKeys.GENERAL_DOWNLOADS_MOBILE, true);
```

Comment:
in this specific situation, should we comment the code? would it help to the next dev to understand why is this returning the reverse? would it even matter?